src/components/pickers/DatePicker.js

```diff
@@ -19,53 +20,32 @@ export default {
       currentMonth: null,
       currentYear: null,
       isSelected: false,
-      isReversing: false
+      isReversing: false,
+      narrowDays: [],
+      firstDayOffset: 0
     }
   },
 
   props: {
+    locale: {
+      type: String,
+      default: 'en-us'
```

Comment:
Here's the code I've been using in vue-simple-calendar, which uses the locale and Intl for the day and month names:

```
return (
	(navigator.languages && navigator.languages.length) ?
		navigator.languages[0]
		: navigator.language || navigator.browserLanguage
	).toLowerCase();
```